onnx/test/cpp/function_context_test.cc

```diff
@@ -181,5 +126,16 @@ TEST(FunctionAPITest, TypeContextTest) {
   check_function(fnProto, checkerCtx, lexicalScope);
 }
 
+TEST(FunctionAPITest, ContextDependentFunctionTest) {
+  ContextDependentFuntionTest();
+}
+
+// Does not test registering schema twice when no exceptions are enabled.
```

Comment:
The two tests here are actually testing different aspects. The first tests a simple function where the parameter is of a fixed type (float), while the second tests a function where the parameter may be one of several types (float or double). The name "RegisterTwiceSchema" may be confusing, but it is meant to register a function called "CustomFunTwice". Does running these two tests cause a problem (such as same function name being registered twice)? If so, I suggest just renaming the functions as "CustomFunTwice1" and "CustomFunTwice2" or use a static boolean to ensure it is registered only once. Thanks!